README.md

```diff
@@ -16,7 +16,7 @@
 This repository hosts Smart Contracts that are deployed on the [OKP4 network](http://okp4.network). But they are compatible with any
 [Cosmos blockchains](https://cosmos.network/) that uses the [CosmWasm](https://cosmwasm.com/) framework.
 
-### 🗄️ Storage oriented Smart Contracts
+## 🗄️ Storage oriented Smart Contracts
```

Comment:
```suggestion
### 🗄️ Storage oriented Smart Contracts
```

It's open to discussion but I would be more inclined to leave this title in the subsection of `Smart Contracts` because it's a kind of smart contracts.